libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -57,7 +57,7 @@ class SRV_Channel {
         k_mount_roll            = 8,            ///< mount roll
         k_mount_open            = 9,            ///< mount open (deploy) / close (retract)
         k_cam_trigger           = 10,           ///< camera trigger
-        k_egg_drop              = 11,           ///< egg drop, deprecated
+        k_tie_down_release      = 11,           ///< tie down release
```

Comment:
Shouldn't re-use here, but allocating a new one will require a patch into master to avoid conflicts.

Reply:
Yeh, happy to stay out of master on this one. I figured this would make it easy to backport.